### DISCUSSION TITLE
Full replacement of the old speedtest tracker

### DISCUSSION THREAD

**Discussion by masterwishx** [alexjustesen/speedtest-tracker#300] [Q&A]
For the full replacement from old to this new amazing speedtest tracker, I should wait for future that can convert old DB to new?
Or manually to convert sqllite DB old to new if it possible?
I think there is some data that not same in both DB and also missing in old DB (like jitter and etc).

**Comment by alexdelprete** [comment]
DB structure is not the same, so an import function is needed. I think it's on the roadmap. So you can switch now and import old data later.

**Reply by alexjustesen** [reply]
Correct and I took some time the other day to map out the next couple of milestones so everyone can have an idea what my priorities are https://github.com/alexjustesen/speedtest-tracker/milestones

**Reply by masterwishx** [reply]
@alexdelprete 
@alexjustesen Thanks will follow it. 
Sure, I saw it in future planned road map,
That's why i asked about it, for now I'm using both, the old one every 10min, and this one every 1h5m. But this is no best choice to use both so waiting for this future to fully replacement the  old one.

**Reply by alexdelprete** [reply]
If you use both you will have sort of "duplicate data", it's better to switch now, and then when migration is ready you can import past data from the old project. If you need to check past data, you can always access the old project, but from the switch date, you will only need the new ST.

**Reply by masterwishx** [reply]
You defently right, Im using this project only for more than one week and very like it.
For old one take a lot time to integrate to influxdbv2 also made guide for it.
So I have for half year in grafana, if I need.

Also need notify to discord threshold, I saw its planned.

But if I switch now and when will be integration
it will be possibly add old database to new or convert old to new and rewrite new one?

**Reply by alexdelprete** [reply]
> it will be possibly add old database to new or convert old to new and rewrite new one?

the import function will import all records from old db to the new one, so you will have the whole history. that's why I advised to start using the new ST now, so you will only need to import old data without duplicating tests running both in parallel.

**Reply by masterwishx** [reply]
OK, Thanks got it
